### PR TITLE
Add Terraform for the 2021-04-19 pipeline

### PR DESCRIPTION
**Pull request by alexwlchan**

### CODE REVIEW

File: reindexer/get_reindex_status.py [hunk 1]
```diff
@@ -9,6 +9,7 @@
 from elasticsearch.exceptions import NotFoundError
 import humanize
 import tabulate
+from tenacity import retry, stop_after_attempt, wait_exponential
```
Comment (jamesgorrie) [inline reindexer/get_reindex_status.py:12]: Does this need to be added as a dependency somewhere?

**Review by jamesgorrie — APPROVED** [review]

File: pipeline/terraform/stack/elastic_pipeline_storage.tf [hunk 2]
```diff
@@ -6,7 +6,7 @@ resource "ec_deployment" "pipeline_storage" {
   name = "catalogue-pipeline-storage-${var.pipeline_date}"
 
   region                 = "eu-west-1"
-  version                = "7.10.2"
+  version                = "7.12.0"
```
Comment (jamesgorrie) [inline pipeline/terraform/stack/elastic_pipeline_storage.tf:9]: I have done a check, and we don't seem to have any breaking problems... but probably worth double checking if you haven't.

File: reindexer/get_reindex_status.py (hunk 1 above)
Comment (alexwlchan) [reply reindexer/get_reindex_status.py:12]: I don't think we have a requirements.txt for this.

File: pipeline/terraform/stack/elastic_pipeline_storage.tf (hunk 2 above)
Comment (alexwlchan) [reply pipeline/terraform/stack/elastic_pipeline_storage.tf:9]: We only use pretty basic Elasticsearch features in pipeline storage (index/get), so I'd be quite surprised if a minor update broke anything.

Comment (jamesgorrie) [reply pipeline/terraform/stack/elastic_pipeline_storage.tf:9]: Ah, great - I suppose any slight fluctuation in querying etc would be OK too.

Comment (alexwlchan) [reply pipeline/terraform/stack/elastic_pipeline_storage.tf:9]: I don't think we even make any queries tbh.

**Pull request merged**